src/worker/ingestion/process-event.ts

```diff
@@ -192,8 +192,15 @@ export class EventsProcessor {
         if (event === '$create_alias') {
             await this.alias(properties['alias'], distinctId, teamId)
         } else if (event === '$identify') {
```

Comment:
Couple of headscratchers:

- We have $create_alias which goes through the same path above. Thoughts on what to do with that?
- We are increasing the number of times we're fetching the person here. It's slightly expensive.
- We're asserting that person with anonymousDistinctId exists in this code path before aliasing, not sure if it's actually correct.

Given that `.alias` already looks up people, it seems instead worth moving this check to `.alias` into the branch that calls mergePeople.

Also let's log something (for now) when going non-anonymous -> non-anonymous.

Reply:
* For `$create_alias` this seems like an explicit request to alias, hence I left this as is. I can look into the expected usage a little more to see if there is anything we should do with this but it seems like this logic should stay as is.
* Yes it is making it more expensive. I considered moving this check into `.alias` where we're already making the db requests, but it seems like if you ask to alias two ids then it should alias them unconditionally.
* We presumably have to check the anonymous user before aliasing as this is what we want to act conditionally on, but perhaps I'm missing something?

I'll move the check into `.alias` and add some logging.

Reply:
also think alias should be left alone. It's an explicit choice to merge.